### PR TITLE
Updating data upload documentation

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: docs/UpdatedData.md [hunk 1]
```diff
@@ -300,4 +300,18 @@ Leveraging the `deviceEvent` type and creating 2 new subTypes with the same stru
...
+  "subType": "warmup",
+  "eventId": "Warm123456789",
+  "duration": { 
+    "value": 3,
+    "units": "hours"
+  },
+  "deviceId": "Id12345",
+  "deviceTime": "2018-02-01T00:00:00",
+  "time": "2020-05-12T08:50:08.000Z",
+  "inputTime": "2020-05-12T08:40:00.000Z",
```
Comment (yvon-dblg) [inline docs/UpdatedData.md:314]: I think we won't have an InputTime for this event

File: docs/UpdatedData.md [hunk 2]
```diff
@@ -300,4 +300,18 @@ Leveraging the `deviceEvent` type and creating 2 new subTypes with the same stru
...
+  "timezoneOffset": 60
```
Comment (yvon-dblg) [inline docs/UpdatedData.md]: I have set a remark before on the previous PR, but ignored.
I think ignoring the timezoneOffset and set timezone instead is a better idea.
timezoneOffset  is useless.

File: docs/UpdatedData.md (hunk 1 above)
Comment (CedricPerotto) [reply docs/UpdatedData.md:314]: But for now inputTime is required

Comment (yvon-dblg) [reply docs/UpdatedData.md:314]: ok

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**